WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -75,6 +75,10 @@
   <% if @competition.event_restrictions? %>
     The <%= "Delegate".pluralize(@competition.delegates.count) %> requested the adoption of <b>event restrictions</b> because "<%= @competition.event_restrictions_reason %>".
```

Comment:
Are these intentionally not internationalized?

Reply:
Yes, I believe all WCAT communication is intended to be in English.